src/app/recherche/recherche.tsx

```diff
@@ -21,6 +21,7 @@ import { MetaCopy } from "@/components/typography/typography";
 import { SearchMovie } from "@/lib/types";
 import {
   TAG_MAP,
+  cleanString,
```

Comment:
I think this import is unnecessary if we adopt the same structure for `TheaterSearchResults` than for `Results`

Reply:
what do you mean ? we also use cleanString in the Results function when we call getFields. So i do think we need it.

Reply:
Bah oui justement on pourrait appeler `getFields` comme dans `Results`. Je comprends pas pourquoi tu recodes à zéro un string matching alors qu'on a beaucoup réfléchi à celui ci `stringMatchFields`

Reply:
mais c'est bon je l'utilise mtnt